Assets/HoloToolkit/Input/README.md

```diff
@@ -59,11 +59,38 @@ Unity camera that has been customized for Holographic development.
 #### InputManager.prefab
 Input system that manages gaze and various input sources currently supported by HoloLens, such as hands and gestures.
 
-This also includes a fake input source that allows you to simulate hand input when in the editor. By default, this can be done by holding Shift (left hand) or Space (right hand), moving the mouse to move the hand and use the left mouse button to tap.
+This also includes a fake input source that allows you to simulate input when in the editor. By default, this can be done by holding Shift (left input source) or Space (right input source), moving the mouse to move the source and using the left mouse button to tap.
+
+#### MixedRealityCamera.prefab
+Camera capabale of rendering for HoloLens and occluded Windows Mixed Reality enabled devices.
+MixedRealityCameraManager.cs exposes some defaults for occluded aka opaque displays Vs HoloLens.
+You can either use the defaults that have been set or customize them to match your application requirements.
+
+**For HoloLens:**
+1. 'Clear Flags' is set to 'Solid Color' and Color to 'Clear' as black renders transparent in HoloLens.
+2. Near clip plane is set to 0.85 per comfort recommendations.
+3. Quality Settings to be Fastest.
+
+**For occluded aka opaque devices:**
```

Comment:
It was created to let you build one appx and deploy it to either kind of device, with the app/camera settings adapting as necessary.